pipelines/internal/commands/export/export.go

```diff
@@ -0,0 +1,201 @@
+// Copyright 2018 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package export provides a sub-tool for exporting pipelines to BigQuery.
+package export
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"flag"
+	"fmt"
+	"time"
+
+	"cloud.google.com/go/bigquery"
+	"cloud.google.com/go/civil"
+	genomics "google.golang.org/api/genomics/v2alpha1"
+)
+
+var (
+	flags = flag.NewFlagSet("", flag.ExitOnError)
+
+	filter      = flags.String("filter", "", "the export filter")
+	datasetName = flags.String("dataset", "", "the dataset to export to which must already exist")
+	tableName   = flags.String("table", "", "the table to export to")
+)
+
+type row struct {
+	Name  string
+	Done  bool
+	Error *status `bigquery:",nullable"`
+
+	// The raw pipeline JSON.
+	Pipeline string
```

Comment:
From our discussion, what about pulling out Resources as a separate JSON blob?  I think we'd also considered dropping the Pipelines blob altogether and having Actions, Resources, Labels.. ?

Reply:
Let's start with this, per our chat.